tests/hawc/apps/riskofbias/test_riskofbias_models.py

```diff
@@ -31,3 +32,34 @@ def test_get_orphan_relations(self):
             """
         ).strip()
         assert actual == expected
+
+
+@pytest.mark.django_db
+class TestRiskOfBias:
+    def test_get_dp_export(self):
+        assessment_id = 1
+        study_id = 1
+        _, scores_map = RiskOfBias.get_dp_export(assessment_id, [study_id], "animal")
+
+        # all of the metrics are mapped
+        set(
+            RiskOfBiasScore.objects.filter(
+                riskofbias__study=study_id, riskofbias__final=True, riskofbias__active=True,
+            ).values_list("metric_id")
+        ) == {metric for _, metric in scores_map.keys()}
+
+        # however, study has more scores than metrics
+        assert RiskOfBiasScore.objects.filter(
+            riskofbias__study=study_id, riskofbias__final=True, riskofbias__active=True,
+        ).count() > len(scores_map.keys())
+
+        # make sure only default scores are used
+        for (_, metric), score in scores_map.items():
+            expected_score = RiskOfBiasScore.objects.get(
```

Comment:
i'm not sure how many queries this might be, but it could potentially be a lot. is there a single query that can get all the data and then you  could build a metric key:value dict to access? might speed it up,